app/scripts/directives/bindService.js

```diff
@@ -55,20 +59,6 @@
       }
     };
 
-
-    var deploymentConfigs, deployments, replicationControllers, replicaSets, statefulSets;
-    var sortApplications = function() {
```

Comment:
I moved it back to `bindService` as it originally was. Def think this is a good call if we end up sharing it in the future.